mmcv/ops/upfirdn2d.py

```diff
@@ -0,0 +1,191 @@
+# modify from https://github.com/rosinality/stylegan2-pytorch/blob/master/op/upfirdn2d.py  # noqa:E501
+
+import torch
+from torch.autograd import Function
+from torch.nn import functional as F
+
+from ..utils import ext_loader
+
+upfirdn2d_ext = ext_loader.load_ext('_ext', ['upfirdn2d'])
+
+
+class UpFirDn2dBackward(Function):
```

Comment:
May add docstrings somewhere to indicate `UpFirDn2d` is which operator's abbreviation.